<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workermainwindow</class>
 <widget class="QDialog" name="workermainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>578</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>981</width>
    <height>578</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>981</width>
    <height>578</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QListWidget" name="list1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>551</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#4d65ea;&quot;&gt;спмсмсм&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Segoe Script&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>Гарячі страви</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Напої</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Десерти</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ланчі</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Піцци</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Салати</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="list2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>301</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Segoe Script&quot;;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>411</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Segoe Script&quot;;</string>
     </property>
     <property name="text">
      <string>Номер замовлення</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Segoe Script&quot;;</string>
     </property>
     <property name="text">
      <string>Ціна замовлення</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Segoe Script&quot;;</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Segoe Script&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe Script&quot;;
background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Зробити замовлення</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>450</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe Script&quot;;
background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Очистити</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe Script&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>251</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe Script&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe Script&quot;;</string>
    </property>
    <property name="text">
     <string>Кількість зароблених грошей працівником</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
